thunder/executors/torchex.py

```diff
@@ -2104,6 +2104,7 @@ def _stash_grad_for_fsdp_prim_impl(
 
 
 has_einops = importlib.util.find_spec("einops") is not None
+has_einops &= importlib.util.find_spec("einops._backends") is not None
```

Comment:
```suggestion
if has_einops:
    importlib.util.find_spec("einops._backends") is not None
```
because `&=` is not shortcutting and errors in docs.

Reply:
Did you mean `has_einops = importlib.util.find_spec("einops._backends") is not None`?

Reply:
I'm stupid, yes. Sorry. :)